src/pages/reports/{MarkdownRemark.fields__slug}.js

```diff
@@ -31,10 +31,14 @@ function ReportPageTemplate({ data }) {
             </h1>
             <h4>{page.frontmatter.date}</h4>
           </div>
-          <div
-            className="report-contents"
-            dangerouslySetInnerHTML={{ __html: DOMPurify.sanitize(page.html) }}
-          />
+          <div className="report-container">
+            <h2>Table of content</h2>
+            <div className="report-toc" dangerouslySetInnerHTML={{ __html: DOMPurify.sanitize(page.tableOfContents) }}/>
```

Comment:
Can you update the layout of the page to have the table of contents on to the side and statically positioned?

Reply:
@captainmangoC4 After our discussion, I think that this is outdated right as I added the button to back to stop :) 

Merging is blocked :-)

Reply:
Yeah, the back to top button is a nice solution.